auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,90 +1906,970 @@ paths:
+        example: "field_name is a required property"
+      exception:
+        description: An human readable explanation specific to this occurrence of the problem. Same as detail. Used in some UI forms.
+        type: string
+        example: "field_name is missing"
+      status:
+        description: The HTTP status code generated by the origin server for this occurrence of the problem.
+        type: integer
+        format: int32
+        minimum: 400
+        maximum: 499
+        example: 400
+      title:
+        description: A short, human-readable summary of the problem type. It SHOULD NOT change from occurrence to occurrence of the problem, except for purposes of localisation.
+        type: string
+        example: "Bad Request"
+      type:
+        description: "An absolute URI that identifies the problem type.  When dereferenced, it SHOULD provide human-readable documentation for the problem type (e.g., using HTML).When this member is not present its value is assumed to be 'about:blank'"
+        type: string
+        example: "about:blank"
+
+  CSRFModel:
+    title: CSRF Token
+    description: CSRF Token in POST/PUT or UPDATE requests
+    type: object
+    required:
+      - csrf_token
+    properties:
+      csrf_token:
+        type: string
+        example: "1491342563##c4e6fef0b978e6c89af9ff1015e67b9ca7c45d14"
+
+  CountModel:
+    title: Count
+    description: count of total objects/elements fetched
+    type: object
+    required:
+      - count
+    properties:
+      count:
+        description: count of total values fetched from DB
+        type: integer
+        minimum: 0
+        format: int32
+        example: 1
+
+  DataVersionModel:
+    title: Data Version
+    description: "model to re-use 'data_version' field."
+    type: object
+    properties:
+      data_version:
+        description: version number of object
+        type: integer
+        format: int32
+        minimum: 1
+        example: 3
+
+  DataVersionNullable:
+    title: Nullable Data Version
+    description: "model to re-use nullable 'data_version' field for scheduled_changes"
+    type: object
+    properties:
+      data_version:
+        description: the version number of object
+        type: ["integer", "null"]
+        format: data_version
+        example: 1
+
+  SCIdModel:
+    title: SC ID
+    description: "model to re-use 'sc_id' field"
+    type: object
+    required:
+      - sc_id
+    properties:
+      sc_id:
+        description: id of scheduled change
+        type: integer
+        format: int32
+        minimum: 0
+        example: 2
+
+  SignoffsRequiredModel:
+    title: Signoffs Required
+    description: "model to re-use 'signoffs_required' integer field."
+    type: object
+    properties:
+      signoffs_required:
         description: The number of signoffs required for a given role
         type: ["integer", "string"]
         format: signoffs_required
         example: 2
 
+  SignoffsRequiredNullable:
```

Comment:
I did see instances of this field getting set as null when operating GET on /scheduled_changes/required_signoffs/permissions or /scheduled_changes/required_signoffs/product. Also , one of the test cases failed when field was set to non-nullable. here's the [test case](https://github.com/aksareen/balrog/blob/ddee69f55db14de816c84d3615426661c3dce846/auslib/test/admin/views/test_required_signoffs.py#L651)